docs/installation.md

```diff
@@ -0,0 +1,210 @@
+# Installing JAX
+
+JAX is written in pure Python, but it depends on XLA, which needs to be
+installed as the `jaxlib` package. Use the following instructions to install a
+binary package with `pip` or `conda`, to use a
+[Docker container](#docker-containers-nvidia-gpu), or to [build JAX from
+source](developer.md#building-from-source).
+
```

Comment:
It may be worth replicating the table from the README here. It's a really nice visual summary, and the table elements could link to the corresponding sections below.